owncloudApp/src/main/java/com/owncloud/android/ui/preview/ViewPagerWorkAround.kt

```diff
@@ -0,0 +1,34 @@
+package com.owncloud.android.ui.preview
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.viewpager.widget.ViewPager
+
+/**
+ * [Workaround](https://github.com/Baseflow/PhotoView/issues/31#issuecomment-19803926)
+ */
+class ViewPagerWorkAround : ViewPager {
+    constructor(context: Context) : super(context)
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {}
+
+    @SuppressLint("ClickableViewAccessibility")
+    override fun onTouchEvent(ev: MotionEvent?): Boolean {
+        try {
+            return super.onTouchEvent(ev)
+        } catch (ex: IllegalArgumentException) {
+            ex.printStackTrace()
```

Comment:
Consider to use `Timber.e(ex)` when you want to see it in logfil. This is more a `/dev/null`

Reply:
Yes, you are right!